easybuild/easyblocks/i/icc.py

```diff
@@ -74,21 +74,28 @@ def install_step(self):
                 'license_file_name': LICENSE_FILE_NAME_2012,
             }
 
-        super(EB_icc, self).install_step(silent_cfg_names_map=silent_cfg_names_map)
+        if LooseVersion(self.version) >= LooseVersion('2016'):
+            cfg_extras_map = {
+                'COMPONENTS': 'ALL',
+            }
+        super(EB_icc, self).install_step(silent_cfg_names_map=silent_cfg_names_map, silent_cfg_extras=cfg_extras_map)
```

Comment:
None that I have found?
See https://software.intel.com/en-us/articles/intel-composer-xe-2015-silent-installation-guide under `COMPONENTS`